### PR TITLE
KEYCLOAK 15634: Add ldap test

### PR DESCRIPTION
**Pull request by vanrar68**
The test submitted along with the fix was for the "generic attribute removal" scenario.
The test in this PR addresses the "LDAP federated user removal" scenario (which was reported in the first place in KEYCLOAK 15634)
Feel free to discard if not useful anymore @mposolda

### CODE REVIEW

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LDAPAccountTest.java [hunk 1]
```diff
@@ -19,15 +19,24 @@
...
+import org.keycloak.testsuite.admin.ApiUtil;
 import org.keycloak.testsuite.federation.ldap.LDAPTestContext;
+import org.keycloak.testsuite.ui.account2.page.PersonalInfoPage;
 import org.keycloak.testsuite.ui.account2.page.SigningInPage;
 import org.keycloak.testsuite.util.LDAPRule;
 import org.keycloak.testsuite.util.LDAPTestUtils;
 
+import java.util.Map;
+
+import static org.junit.Assert.*;
```
Comment (mposolda) [inline testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LDAPAccountTest.java:39]: Is it possible to remove star import?

**Review by mposolda — CHANGES_REQUESTED** [review]
@vanrar68 Thanks! The PR looks good to me. Just added some minor detail in the comment...

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LDAPAccountTest.java (hunk 1 above)
Comment (vanrar68) [reply testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LDAPAccountTest.java]: Sure, done!

**Review by mposolda — APPROVED** [review]
@vanrar68 Thanks!

**Pull request merged**